### PR TITLE
Added computation of the FID score

### PR DESCRIPTION
**Pull request by andimarafioti**
Adaptation of score.py to also compute the Fréchet Inception Distance using code provided by the authors of the measure.
It makes sense to add it side to side to the evaluation of the inception measure since the two don't correlate precisely but are built on the same classifier network. In the future, it would be nice to have cleaner code and a better documentation. As it is, there is a lot of repetition between this file and score.py, but I think it's better to push it now than wait until it's perfect.

### CODE REVIEW

File: eval/inception/score_and_FID.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    graph = tf.Graph()
+    with graph.as_default():
+        saver = tf.train.import_meta_graph(metagraph_fp)
+        
+        if use_tf_ffmpeg:
+            x_fp = tf.placeholder(tf.string, [])
+            x_bin = tf.read_file(x_fp)
+            x_samps = tf.contrib.ffmpeg.decode_audio(x_bin, tf_ffmpeg_ext, 16000, 1)[:, 0]
+    x = graph.get_tensor_by_name('x:0')
+    activations = graph.get_tensor_by_name('classifier/dense/BiasAdd:0')
```
Comment (AlexeyG) [inline eval/inception/score_and_FID.py]: It looks like you are taking the logits (a 10-D vector) and compute the FID score on them. Shouldn't this be the activations of one of the earlier pooling layers (e.g. the pool4 layer) as in the original FID score?

However, one issue that would arise if an earlier layer is taken is that the dimensionality of the activations would be too high (it is already 8192 for the pool4 layer) to compute the FID score in a reasonable time. This is because complexity of the matrix square root computation is cubic in the matrix dimensionality.
